### DISCUSSION TITLE
Incremental Deployment : for existing adf

### DISCUSSION THREAD

**Discussion by sandeeppatidar76** [Azure-Player/azure.datafactory.tools#368] [General]
@NowinskiK, Can we deploy new objects using incremental deployment in existing adf(already deployed few objects), i have tried it but it giving error as statefile not found, so can please provide some steps how can we use incremental deployment for our purpose.

**Comment by NowinskiK** [comment]
Yes, you can. This is how it works. You need to enable IncrementalDeployment flag only.
If you faces any error within the feature - raise an issue with appropriate details.
